.github/workflows/stale.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           stale-pr-label: "stale"
           stale-issue-message: "Hi, @gigincg, @nihal467, @khavinshankar, @mathew-alex, @aparnacoronasafe, This issue has been automatically marked as stale because it has not had any recent activity."
           stale-pr-message: "Hi, This pr has been automatically marked as stale because it has not had any recent activity. It will be automatically closed if no further activity occurs for 7 more days. Thank you for your contributions."
-          close-pr-message: "Hi, @gigincg, @nihal467, @khavinshankar, @mathew-alex, This pr has been automatically closed because it has not had any recent activity. Thank you for your contributions. Feel free to repopen the pr."
+          close-pr-message: "Hi, @gigincg, @nihal467, @khavinshankar, @mathew-alex, This PR has been automatically closed due to inactivity. Thank you for your contributions. Feel free to re-open the PR."
```

Comment:
@rithviknishad also can you change the users names to team, something like @coronasafe/care-developers